lib/libutee/include/user_ta_header.h

```diff
@@ -30,73 +30,30 @@
 
 #include <tee_api_types.h>
 
+/* XXX */
+#define TA_FLAG_USER_MODE 0
```

Comment:
All these TAs are user mode. We should probably remove the define completely as we're breaking backwards compatibility with this commit any way.